src/frontend/staff-portal/src/app/components/paging/paging.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="main-pagination">
+    <div class="pagination">
+        <a [class.disabled]="currentPage === 1" (click)="changePage(currentPage - 1)"><<</a>
+        <ng-container *ngFor="let e of [].constructor(totalPages); let i = index">
+            <a [class.active]="i+1 === currentPage" (click)="changePage(i+1)" *ngIf="showPage(i+1)">{{i+1}}</a>
```

Comment:
Why we are not using mat-paginator, it also handled accessibility stuff.
https://material.angular.io/components/paginator/overview

Reply:
Made the custom pagination to align with the BC Gov design standard
https://bcgov.github.io/bootstrap-theme/docs/reference/simple/

Reply:
I will do this change

Reply:
Done